internal/service/project/resource_billing_group.go

```diff
@@ -91,6 +91,12 @@ var aivenBillingGroupSchema = map[string]*schema.Schema{
 		DiffSuppressFunc: schemautil.EmptyObjectNoChangeDiffSuppressFunc,
 		Description:      "State",
 	},
+	"copy_from_billing_group": {
```

Comment:
Since this field can only be setduring the creation of a resource please add `DiffSuppressFunc: schemautil.CreateOnlyDiffSuppressFunc,` and since it cannot change after creation we do not need `schemautil.EmptyObjectNoChangeDiffSuppressFunc`